AnnoyancesFilter/sections/popups.txt

```diff
@@ -81,6 +81,7 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+||*jquery.fancybox.min.js$domain=itemci.com
```

Comment:
I don't recommend to block this script. It can be used for useful purposes. Is this rule required?
Also don't use redundant part `||*`. `||` cover domain + sub-domains, `*` any character.